release/example/str.trie_sencoten/source/example.str.trie_sencoten.model.kps

```diff
@@ -27,7 +27,8 @@
       <ID>example.str.trie</ID>
       <Version>0.1.0</Version>
```

Comment:
I think this version might need to bump to 0.1.1 too

Reply:
Version numbering may still be a bit WET. Working on reducing that in the future...

Reply:
Dumb question: why is there a separate file version and a lexical model version. Should they not be the same?

Reply:
Technically, the lexical model is developed independently to the package that contains it. In this repo, it makes sense for the .model.kps version to follow the lexical model version precisely... we have that functionality available with the `<FollowKeyboardVersion />` property in the `<Options>` key -- which is badly named but too late to fix.

Looking at the compiler, it understands that property but does not yet use it ... so that's something we need to implement.